modules/instance_template/main.tf

```diff
@@ -204,6 +209,7 @@ resource "google_compute_instance_template" "tpl" {
 
   confidential_instance_config {
     enable_confidential_compute = var.enable_confidential_vm
+    confidential_instance_type  = var.confidential_instance_type
```

Comment:
Looks like this was added in https://github.com/hashicorp/terraform-provider-google/releases/tag/v5.36.0.  Please bump the minimum here: https://github.com/terraform-google-modules/terraform-google-vm/blob/master/modules/instance_template/versions.tf#L22

Reply:
Done, thanks!